etc/meld.profile

```diff
@@ -22,6 +22,7 @@ noblacklist ${HOME}/.ssh
 noblacklist ${HOME}/.subversion
 
 # Allow python (blacklisted by disable-interpreters.inc)
+# Python 2 is EOL (see #3164). Uncomment the next line (or put it into your meld.local) if you understand the risks but want python 2 support for older meld versions.
```

Comment:
```suggestion
# Python 2 is EOL (see #3164). Uncomment the next line (or put it into your meld.local) if you understand the risks but want python 2 support for older meld versions.
#include allow-python2.inc
```

Reply:
Hehe, my subconscious python2 love showed itself... Nice catch, and more importantly, nice job proposing this in #3164 in the first place :+1: